app/scripts/stores/polygon.es6

```diff
@@ -0,0 +1,20 @@
+import { createStore } from 'reflux';
+import SaneStore from '../utils/sane-store-mixin';
+import { loadRegionsCompleted, loadDistrictsCompleted } from '../actions/polygon';
+
+const PolygonsStore = createStore({
+  initialData: [],
```

Comment:
`initialData` is an array but `setData` makes it an object -- this should probably be `{}`.